auditors/capabilities/fixtures/capabilities-redundant-override.yml

```diff
@@ -11,25 +11,12 @@ spec:
     metadata:
       labels:
         name: deployment
-        audit.kubernetes.io/pod.allow-capability-chown: "SomeReason"
+        audit.kubernetes.io/pod.allow-capability-all: "SomeReason"
```

Comment:
I don't think we want anyone to ever override the `all` capability. Even though there's nothing in the code stopping someone from doing this, I don't think we should make it an example. Since there's nothing in the add list, any capability override is redundant, so I think keeping it as `audit.kubernetes.io/pod.allow-capability-chown` should be fine